### PR TITLE
I added a bio about myself to README.md

### PR DESCRIPTION
**Pull request by benjaviv**
I edited the README.md file with a brief bio about myself and my interests. I included bold and italic text, a list, headers of various sizes, as well as a link to the University of Chicago website. While editing the text was somewhat straightforward, it was not without its difficulties. I had difficulty finding a way to preview the markdown text without consistently committing to GitHub, and having to commit in order to view the result of every change in r studio. I ended up using a markdown preview website to quickly check minor changes. 

In addition, I had difficulty uploading an image to the markdown, and will attempt this in a future project. I also had difficulty adding and controlling line breaks, but ended up figuring out how to control line breaks by adding two spaces after each line.

### CODE REVIEW

**Review by rkcatipon — COMMENTED** [review]
Hi Ben! Thanks for sharing your code. Here is some feedback:
For next time:
- Did you want the UChicago link to aligned with the unordered list?
- Bolding your name in the H2 header won't be visible since the header is already bolded.
- Maybe link the first time you mention the University of Chicago.

A few wins:
- Good use of bolded and italicized text!
- Included a nice example of a list.
- Your headers adhere to a clear hierarchy.

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# hw01
\ No newline at end of file
+# HW 01
```
Comment (nearridge) [inline README.md:1]: Standard R-markdown practice is to empty lines between your markdown elements so there should be a return after each line beginning with a # if there isn't one already.

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+## A Brief Bio of **Ben Weinstein**
+
+My name is **Ben Weinstein**. I am a 4th year at the *University of Chicago*. I am new to **computer languages**, although I do like **foreign languages**.
+Some of the foreign languages I have studied are:  
+- Mandarin    
+- Arabic    
+- Spanish  
+[The University of Chicago](https://www.uchicago.edu)
```
Comment (nearridge) [inline README.md:9]: There is a missing bullet point here.

File: README.md (hunk 1 above)
Comment (nearridge) [inline README.md:1]: Standard R Markdown practice is to have a return after each markdown element (headers, code chunks, plots etc.). That'll help space things out and will help you avoid any rendering issues later on.

**Review by nearridge — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |           |         x     |            |
| **Reproducibility**         |      x     |              |            |

Remarks: Great job here, especially using git. It's definitely not the most obvious thing but it looks like you had no trouble using it towards the end. We'll talk about how you can preview the rendered output of markdown files soon. If you have any questions in the meantime, feel free to come to office hours and ask about "knitting" a file.

**Comment by tkymtsmt** [comment]
Hi Ben! Thanks for sharing your code.


Here is a few feedback:

- Good points
    - the usage of unordered list.
    - the usage of **bold** and _italic_ 

- Something you may improve (kinda tips)
    - The UChicago link is not aligned with the unordered list. 
    - You draw lines to split categories. To do so, you can use 
         - backslash, `\` to add space lines as well as 
         - headers, `#`.

**Pull request closed without merging**